src/api/files.py

```
@@ -0,0 +1,88 @@
+
+from fastapi import Depends, Query
+from fastapi.responses import JSONResponse
+
+from dependencies import get_session_manager, get_current_user
+from session_manager import User
+from utils.logging_config import get_logger
+
+logger = get_logger(__name__)
+
+
+def _get_file_service(session_manager=Depends(get_session_manager)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)